From: Miguel Vadillo <vadillo@ti.com>
Date: Wed, 16 Feb 2011 00:03:25 +0000 (-0600)
Subject: syslink: ipu_pm: remove gptimer 9 as requestable
X-Git-Tag: android-2.6.35-2.3-omap4.13.1~85
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=b04e7ad2e5b20cf6209919463c24dbe7d7c33126

syslink: ipu_pm: remove gptimer 9 as requestable

gptimer 4 and 9 are used by IPU as ticks for both
M3 cores thus remove it from the list of requestable
resources by IPU resource manager.

Change-Id: I228bf5a656508c89c8c6deaa02404fc8359d75b1
Signed-off-by: Miguel Vadillo <vadillo@ti.com>
---

diff --git a/drivers/dsp/syslink/ipu_pm/ipu_pm.c b/drivers/dsp/syslink/ipu_pm/ipu_pm.c
index f8d6c6d..22e01c8 100644
--- a/drivers/dsp/syslink/ipu_pm/ipu_pm.c
+++ b/drivers/dsp/syslink/ipu_pm/ipu_pm.c
@@ -341,7 +341,6 @@ static void __iomem *fdifHandle;
 static struct clk *clk_opt_iss;
 /* Ducati Interrupt Capable Gptimers */
 static int ipu_timer_list[NUM_IPU_TIMERS] = {
-	GP_TIMER_9,
 	GP_TIMER_11};
 
 /* I2C spinlock assignment mapping table */
diff --git a/drivers/dsp/syslink/ipu_pm/ipu_pm.h b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
index 787ecc6..121c7e9 100644
--- a/drivers/dsp/syslink/ipu_pm/ipu_pm.h
+++ b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
@@ -128,7 +128,7 @@
 #define GP_TIMER_4 4
 #define GP_TIMER_9 9
 #define GP_TIMER_11 11
-#define NUM_IPU_TIMERS 2
+#define NUM_IPU_TIMERS 1
 
 #define I2C_SL_INVAL -1
 #define I2C_1_SL 0
